src/modules/Builder/api/api.types.ts

```diff
@@ -66,3 +67,24 @@ export type LorisUsersVisit = {
 export type UploadLorisUsersVisitsParams = AppletId & {
   payload: LorisUsersVisit[];
 };
+
+export type FetchIntegrationStatusParams = {
+  // currently set as snake case on the API
+  applet_id: string;
+  integration_type: IntegrationTypes;
```

Comment:
Could you double check this please, in reviewing the backend response I see camelCase responses
![CleanShot 2024-10-11 at 12 16 29@2x](https://github.com/user-attachments/assets/af5cb439-0784-4d3e-b001-db8d6445c940)

Reply:
Thanks, confirmed the GET request expects snake case.